### PR TITLE
add JetBrains.Toolbox

### PR DESCRIPTION
**Pull request by VagyokC4**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1679)

### CODE REVIEW

File: manifests/JetBrains/Toolbox/1.17.7018.0.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Id: JetBrains.Toolbox
+Publisher: Jet Brains
```
Comment (splitt3r) [inline manifests/JetBrains/Toolbox/1.17.7018.0.yaml:2]:
```suggestion
Publisher: JetBrains
```

They always use `JetBrains` (see https://www.jetbrains.com/company/).

**Review by splitt3r — CHANGES_REQUESTED** [review]

**Review by SoftCreatR — CHANGES_REQUESTED** [review]
Publisher Name needs to be fixed: `Jet Brains` => `JetBrains`

EDIT: Nvm. Too slow 😄

**Review by ghost — APPROVED** [review]
Validation has completed

**Comment by MarNicGit** [comment]
@jamierocks All I'm reading is that you're making assumptions without actually testing out what you're claiming. A quick `winget install jetbrains.toolbox` shows that it installs perfectly fine without any user interaction whatsoever.

**Comment by jamierocks** [comment]
I'm relying on https://github.com/KevinLaMS's original statement that this was an issue, this manifest included nothing new - so i would assume the same blocking-issue is still in place.

**Pull request merged**